src/bp/ui-studio/src/web/views/OneFlow/diagram/VariableTypesForm/EnumForm.tsx

```diff
@@ -75,6 +78,28 @@ const EnumForm: FC<Props> = ({
     updateFormItem(convertFromTags(data))
   }
 
+  const isDupplicate = (localItem: Item[], newItem: Item) => {
```

Comment:
Typo "dupplicate". You have an array, use the plural `items: Item[]`

Reply:
Changed typo, and changed localItem to localItems (to keep consistency with TagInputList)

Reply:
Technically your method is in "EnumForm" and not TagInputList, so it's not necessary. Keep short names when possible